src/cloud-api-adaptor/test/e2e/libvirt_test.go

```diff
@@ -96,3 +97,12 @@ func TestLibvirtPodsMTLSCommunication(t *testing.T) {
 	assert := LibvirtAssert{}
 	DoTestPodsMTLSCommunication(t, testEnv, assert)
 }
+
+func TestLibvirtKbsKeyRelease(t *testing.T) {
+	if os.Getenv("DEPLOY_KBS") == "false" || os.Getenv("DEPLOY_KBS") == "no" {
```

Comment:
Should we make this a positive test to match the set-up logic e.g.
```
if ! (os.Getenv("DEPLOY_KBS") == "true" || os.Getenv("DEPLOY_KBS") == "yes")
```
?

Reply:
Revised.